deny.toml

```diff
@@ -80,9 +80,18 @@ yanked = "warn"
 # A list of advisory IDs to ignore. Note that ignored advisories will still
 # output a note when they are encountered.
 # https://embarkstudios.github.io/cargo-deny/checks/advisories/cfg.html#the-ignore-field-optional
-ignore = [
-    { id = "RUSTSEC-2023-0071", reason = "Currently no fix, but it's currently being worked on, see issue: https://github.com/RustCrypto/RSA/issues/19#issuecomment-1822995643" },
-]
+[[ignore]]
+id = "RUSTSEC-2023-0071"
+reason = "Currently no fix, but it's currently being worked on, see issue: https://github.com/RustCrypto/RSA/issues/19#issuecomment-1822995643"
+
+[[ignore]]
+id = "RUSTSEC-2024-0363"
+reason = """
+Possible SQL Injection where input manipulate the SQL query by overflowing the buffer len field (4 bytes).
+
+We are not affected since sqlx is used on the client side to store the files' data.
```

Comment:
On top of that, the issue seems to impact PostgreSQL where we only use sqlx for SQLite (see the screenshot calling `pgio.SetInt32` in https://github.com/launchbadge/sqlx/issues/3440).

I've asked for confirmation https://github.com/launchbadge/sqlx/issues/3440#issuecomment-2296009818